### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by riyazdf**
Fixes to the docs from feedback in #603

### CODE REVIEW

File: docs/reference/client-config.md [hunk 1]
```diff
@@ -21,38 +20,25 @@ but the path to a different configuration file can be specified using the
...
+JSON keys to learn more about the configuration section corresponding to that key:
 
-In addition please see the optional password [environment
-variables](#environment-variables-optional) that the Notary client can take for
-ease of use.
 
-An example (full) clietn configuration file.
-
-```json
+<pre><code class="language-json">
```
Comment (endophage) [inline docs/reference/client-config.md]: have we confirmed the markdown compiler will handle this tag correctly?

Comment (riyazdf) [inline docs/reference/client-config.md]: yup, it looks pretty good when I generate the docs :)

screen shot 2016-03-01 at 3 10 31 pm

Comment (endophage) [inline docs/reference/client-config.md]: :+1: nice

Comment (endophage) [inline docs/reference/client-config.md]: Although, looks like it's including the initial newline at the end of the tag. Want to try moving the `{` to immediately after the opening `<code>` tag?

File: docs/service_architecture.md [hunk 2]
```diff
@@ -104,53 +103,57 @@ sever, and signer:
...
-    attacks.
+    attacks, so long as the attacker ensures that the version number of their
+    malicious metadata is higher than the version number of the most recent
+    good metadata that any client may have.
 
- Note that the timestamp and snapshot keys are never compromised; once the
- Server compromise is mitigated, an attacker will not be able to generate
- valid timestamp or snapshot metadata and serve them on a malicious mirror, for
- example.
+ Note that the timestamp and snapshot keys cannot be compromised in a server-only
```
Comment (endophage) [inline docs/service_architecture.md]: @cyli saw your comment on the previous docs PR about this. It read a bit too much like a statement in the original version, which was a little confusing. The extra context here makes the meaning clearer.

**Comment by endophage** [comment]
Just the one newline to be fixed then rebase and LGTM.

File: docs/reference/client-config.md (hunk 1 above)
Comment (riyazdf) [inline docs/reference/client-config.md]: good catch, done!

File: docs/service_architecture.md (hunk 2 above)
Comment (cyli) [inline docs/service_architecture.md]: Ah ok, cool thanks.  Did we want to move the threat model stuff into another doc page as you suggested in the previous PR?

Comment (riyazdf) [inline docs/service_architecture.md]: I'm fine with this as is -- the threat model makes much more sense with the architecture explanation as prior context.

Comment (endophage) [inline docs/service_architecture.md]: Yeah, Diogo and I also agreed that this was good to go as is. At a later point we might want to write up a more detailed and exhaustive threat model and at that point we would separate it out.

**Pull request merged**